common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -173,13 +181,47 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
     result.getElementsByTag("amp-vimeo").size should be(0)
   }
 
-  "AmpEmbedCleaner" should "be able to create an amp-youtube element and an amp-vimeo in the same document" in {
+  "AmpEmbedCleaner" should "replace an iframe in a http Facebook video-element with an amp-facebook element" in {
```

Comment:
It was testing for http facebook embeds. Buuut looking at facebook again, I don't think its possible to embed a facebook video as http - not through our tools anyway. Stripping it out.